examples/example-vite-grpc-playground/src/components/query-results.tsx

```
@@ -0,0 +1,107 @@
+                                    {Object.keys(response[0]).map((header) => (
+                                        <th
+                                            key={header}
+                                            className="px-6 py-3 text-left text-xs font-medium text-zinc-400 uppercase tracking-wider"
+                                        >
+                                            {header}
+                                        </th>
+                                    ))}
+                                </tr>
+                            </thead>
+                            <tbody className="divide-y divide-zinc-800">
+                                {response.map((row: any, rowIndex: number) => (
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)